vendors/halo2/include/bn254_proving_key.h

```diff
@@ -17,7 +17,7 @@ class Prover;
 
 class ProvingKey {
  public:
-  ProvingKey(uint8_t ls_type, rust::Slice<const uint8_t> pk_bytes);
```

Comment:
This appears to reset [this commit](https://github.com/kroma-network/tachyon/pull/464/commits/fa30fa21b6a2af22ce92b7b4dcb2316f3a4cb155) - is this intentional?

Reply:
hmm, I can make it better, I'll organize this